sdk/src/main/java/io/dapr/actors/DaprClientBuilder.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors;
+
+import okhttp3.OkHttpClient;
+
+/**
+ * Builds an instance of DaprAsyncClient or DaprClient.
+ */
+public class DaprClientBuilder {

```

Comment:
Doesn't need to be public, this will used by other public Apis in SDK. eg. ActorProxy on client side & ActorRuntime class.

Reply:
For now, made it visible to package. If it needs to be accessed by another Java package (not jar) it will probably need to be made public again.